man/git-sync.1.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Is this file generated by `make -C .. man/git-sync.{1,html}`? The change should be written into `git-sync.html` but not the `git-sync.1.html`. It may be a bug but I could not reproduce it in my side...

Reply:
Done, it my fault actually.